### DISCUSSION TITLE
Articles and tutorials about NeuralProphet

### DISCUSSION THREAD

**Discussion by ourownstory** [ourownstory/neural_prophet#205] [Show and tell]
Feel free to share any helpful articles and tutorials that you encounter on the internet.

**Comment by ourownstory** [comment]
[High-level comparison of Prophet and NeuralProphet](https://alec-delany.medium.com/neural-prophet-bridging-the-gap-between-accuracy-and-interpretability-1ab951105076)

**Comment by ourownstory** [comment]
[Simple use case example: Mean Temperature in Deli](https://towardsdatascience.com/neural-prophet-a-time-series-modeling-library-based-on-neural-networks-dd02dc8d868d)

**Comment by ourownstory** [comment]
[Hands-on Comparison of Prophet and NeuralProphet](https://towardsdatascience.com/facebooks-prophet-deep-learning-neuralprophet-76796aed1d86)

**Comment by ourownstory** [comment]
[Using NeuralProphet in R with reticulate](https://amiles.netlify.app/2020/12/calling-neuralprophet-from-r-adventures-in-reticulate.en-us)

**Comment by ourownstory** [comment]
[Simple use case example: Stock Market Predictions](https://medium.com/datadriveninvestor/stock-market-predictions-with-neural-prophet-a-deep-learning-approach-a418fa9dab27)
Note: training data seems to only include weekly dates (when the stock market is open), but predictions include weekends. Thus, the weekly seasonality is random on Sat/Sun. Would need to pass a filter over the future `df` before passing it to `predict`, or just filter the predictions `forecast` before plotting

**Comment by gaurav1195** [comment]
I have not seen an article using the lagged regressors, if somebody has come across any articles can you please share?
